Vagrantfile

```diff
@@ -0,0 +1,106 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
```

Comment:
It would be nice to add some brief documentation to your Readme app as to the fact that this exists, what dependencies it has, and how it can be run